libs/process/opts.go

```diff
@@ -0,0 +1,51 @@
+package process
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+)
+
+type execOption func(*exec.Cmd) error
+
+func WithEnv(key, value string) execOption {
+	return func(c *exec.Cmd) error {
+		if c.Env == nil {
+			c.Env = os.Environ()
```

Comment:
We should pull the env through lib/env such that we can run parallel tests with anything using libs/process.

Reply:
good one